content/en/docs/reference/flow-control/_index.md

```diff
@@ -0,0 +1,181 @@
+  workload-low,      service-accounts, 14,         1,                   system:serviceaccount:default:loadtest, 2023-07-18T00:12:51.487092539Z, 10,           0,          0s,                0001-01-01T00:00:00Z
+  ```
+
+  You can get a more detailed listing with a command like this:
+
+  ```shell
+  kubectl get --raw '/debug/api_priority_and_fairness/dump_requests?includeRequestDetails=1'
+  ```
+
+  The output will be in CSV and similar to this:
+
+  ```none
+  PriorityLevelName, FlowSchemaName,   QueueIndex, RequestIndexInQueue, FlowDistingsher,                        ArriveTime,                     InitialSeats, FinalSeats, AdditionalLatency, StartTime,                      UserName,                               Verb,   APIPath,                                   Namespace,   Name,   APIVersion, Resource,   SubResource
+  exempt,            exempt,           -1,         -1,                  ,                                       2023-07-15T04:51:25.596404345Z, 1,            0,          0s,                2023-07-15T04:51:25.596404345Z, system:serviceaccount:system:admin,     list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
+  workload-low,      service-accounts, 14,         0,                   system:serviceaccount:default:loadtest, 2023-07-18T00:13:08.986534842Z, 10,           0,          0s,                0001-01-01T00:00:00Z,           system:serviceaccount:default:loadtest, list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
+  workload-low,      service-accounts, 14,         1,                   system:serviceaccount:default:loadtest, 2023-07-18T00:13:09.086476021Z, 10,           0,          0s,                0001-01-01T00:00:00Z,           system:serviceaccount:default:loadtest, list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
+  ```
+
+  Explanation for selected column names:
+  - `QueueIndex`: The index of the queue. It will be -1 for priority levels
+    without queues.
+  - `RequestIndexInQueue`: The index in the queue for a given request. It will
+    be -1 for executing requests.
+  - `InitialSeats`: The number of seats will be occupied during the initial
+    (normal) stage of execution of the request.
+  - `FinalSeats`: The number of seats will be occupied during the final stage
+    of request execution, accounting for the associated WATCH notifications.
+  - `AdditionalLatency`: The extra time taken during the final stage of request
+    execution. FinalSeats will be occupied during this time period. It does not
+    mean any latency that a user will observe.
+  - `StartTime`: The time a request starts to execute. It will be
+    0001-01-01T00:00:00Z for queued requests.
+
+## Debug logging
+
+At `-v=3` or more verbosity, the API server outputs an httplog line for every
```

Comment:
httplog is a new term to me ...

Reply:
It is provided by https://github.com/kubernetes/apiserver/blob/master/pkg/server/httplog/doc.go. 
It allows you to add key-value pairs which is heavily used here. https://github.com/kubernetes/apiserver/blob/83bd32f90ba0c7cb082e2605a21e1a9f183481cd/pkg/server/httplog/httplog.go#L227